rust/apple-client-ffi/src/lib.rs

```diff
@@ -301,6 +301,7 @@ impl WrappedSession {
         let (session, mut event_stream) = Session::connect(
             Arc::new(socket_factory::tcp),
             Arc::new(socket_factory::udp),
+            Default::default(), // TODO: Load records from cache.
```

Comment:
I've left TODOs in the places where we need to integrate with the client apps to actually make use of this feature.